### ISSUE TITLE
Multiple batch errors

### ISSUE THREAD

**Issue by calebshannon** [simple-odata-client/Simple.OData.Client#882]
If there are multiple errors returned in my batch response, is it possible to get them all?

I see this in AssignBatchActionResultsAsync():
```
if (exceptions.Any())
{
    throw exceptions.First();
}
```

Am I correct in my understanding that if there are more errors, I will not get them from my ODataBatch?

I'm a bit of a junior so please help me understand it if I'm not thinking about it the correct way.

**Comment by marco10507** [comment]
Hi @calebshannon ,
Did you manage to capture all the errors from the batch? I would like to get all the errors and handle them on my side.

**Comment by calebshannon** [comment]
> Hi @calebshannon , Did you manage to capture all the errors from the batch? I would like to get all the errors and handle them on my side.

No, I ended up not using an OData client library for my project.

**Comment by marco10507** [comment]
@calebshannon , are you using the autogenerated odata client?

https://learn.microsoft.com/en-us/odata/client/code-generation-tool
